tests/test_e2e.py

```diff
@@ -61,6 +62,35 @@ def test_kmeans_summary():
     validate_run(run, log_charts=True)
 
 
+@pytest.mark.filterwarnings("error::neptune.common.warnings.NeptuneUnsupportedType")
+def test_unsupported_object():
+    """This method checks if Neptune throws a `NeptuneUnsupportedType` warning if expected metadata
+    is not found or skips trying to log such metadata"""
+
+    from sklearn.model_selection import GridSearchCV
```

Comment:
Why do we make this import dynamic, instead of global?

Reply:
`GridSearchCV` is needed only in this test, so I thought there was no point importing it globally. But I can move it to the top of the script if that's best-practice.

Reply:
Yes, that would be better